CMakeLists.txt

```diff
@@ -92,6 +92,11 @@ endforeach()
 set(${PROJECT_NAME}_VERSION ${PYBIND11_VERSION_MAJOR}.${PYBIND11_VERSION_MINOR}.${PYBIND11_VERSION_PATCH})
 message(STATUS "pybind11 v${${PROJECT_NAME}_VERSION}")
 
+option (USE_PYTHON_INCLUDE_DIR "Install pybind11 headers in Python include directory" OFF)
```

Comment:
The comment could be more explanatory: "Install [..] within the Python include directory instead of the default installation prefix"